tests/Tests/100__deploy/100__installation/102__post_install.robot

```diff
@@ -18,8 +18,24 @@ Suite Setup         RHOSi Setup
 Resource            ../../../Resources/Page/HybridCloudConsole/HCCLogin.robot
 Resource            ../../../Resources/Common.robot
 
-
 *** Test Cases ***
+Verify Dashbord has no message with NO Component Found
+    [Tags]  Sanity
+    ...     Tier1
+    ...     ODS-1493
+    [Documentation]   Verify "NO Component Found" message dosen't display
+    ...     on Rhods Dashbord page with bad subscription present in openshift
+    [Setup]   Test Setup For Rhods Dashboard
+    Oc Apply  kind=Subscription  src=tests/Tests/100__deploy/100__installation/bad_subscription.yaml
+    Oc Delete    kind=Pod     namespace=redhat-ods-applications    label_selector=app=rhods-dashboard
+    OpenShiftLibrary.Wait For Pods Status    namespace=redhat-ods-applications  label_selector=app=rhods-dashboard  timeout=120
```

Comment:
can this become a kw? `Delete Dashboard Pods And Wait Them To Be Back` for example

Reply:
ack